src/components/Navigator/NavigatorCard.vue

```diff
@@ -82,37 +83,37 @@
             </p>
           </div>
         </div>
-      </NavigatorCardInner>
-    </div>
-    <div class="filter-wrapper" v-if="!errorFetching">
-      <div class="navigator-filter">
-        <div class="input-wrapper">
-          <FilterInput
-            v-model="filter"
-            :tags="availableTags"
-            :selected-tags.sync="selectedTagsModelValue"
-            placeholder="Filter"
-            :should-keep-open-on-blur="false"
-            :position-reversed="isFilterReversed"
-            :clear-filter-on-tag-select="false"
-            class="filter-component"
-            @clear="clearFilters"
-          />
-        </div>
-        <div
-          class="magnifier-icon"
-          @click="store.toggleShowQuickNavigationModal()"
-          v-if="enableQuickNavigation"
-        >
-          <MagnifierIcon />
+        <div class="filter-wrapper" v-if="!errorFetching">
+          <div class="navigator-filter">
+            <div class="input-wrapper">
+              <FilterInput
+                v-model="filter"
+                :tags="availableTags"
+                :selected-tags.sync="selectedTagsModelValue"
+                placeholder="Filter"
+                :should-keep-open-on-blur="false"
+                :position-reversed="!reverseFilterPosition"
```

Comment:
Why is `position-reversed` the opposite to `reverseFilterPosition`. Should it be the same value?
Maybe it's the naming what I find confusing

Reply:
I needed a moment there 🤣 

In the `FilterInput` the default location of the dropdown is below, but for the Navigator, we render the filter down at the bottom, so we need the dropdown to render above.

But on mobile, the filter goes to the top of the `Navigator`, which means the `FilterInput` should render the dropdown in it's default position, which is down.

Reply:
Best case we rename `reverseFilterPosition` to `moveFilterToTop`, `renderFIlterAbove` or something else?